### DISCUSSION TITLE
Support line numbering for multi-file printing

### DISCUSSION THREAD

**Discussion by kuainx** [PDConSec/vsc-print#233] [Ideas]
I right-click the folder in Explorer in vsc and click the Print button, and Line Numbers doesn't show.
I tried to set Print: Line Numbers with on or inherit, it can control the single file line numbers, but can't work with folder.
thank you
![微信截图_20230522153328](https://github.com/PDConSec/vsc-print/assets/22209650/fc94e240-95f8-4f34-8b59-c675ebf1a836)

**Comment by PeterWone** [comment]
Line numbering is not currently supported when multiple files are merge into a single print job.

**Comment by PeterWone** [comment]
Just after moving this to the Ideas discussion it occurred to me how to implement it easily - and it works. Support for line numbering of source code files in multi-file print jobs will appear in the next release. Be aware that plain text files (`*.txt`) are not treated as source code and will not be numbered.

**Comment by kuainx** [comment]
Thanks for your work! :)
